counterpartylib/lib/messages/issuance.py

```diff
@@ -133,7 +176,10 @@ def validate (db, source, destination, asset, quantity, divisible, callable_, ca
             balances = cursor.fetchall()
             cursor.close()
             if util.enabled('numeric_asset_names'):  # Protocol change.
-                if len(asset) >= 13:
+                if subasset_longname is not None and util.enabled('subassets'): # Protocol change.
```

Comment:
My comment here would be in regards to this check: ``` if asset[0] != 'A': ```. Just to point out that if, in the future the 'A' namespace is opened up, this logic would fail.